homeassistant/components/zha/core/registries.py

```diff
@@ -62,27 +63,28 @@
         zigpy.profiles.zll.DeviceType.SCENE_CONTROLLER,
     ],
 }
+REMOTE_DEVICE_TYPES = collections.defaultdict(list, REMOTE_DEVICE_TYPES)
 
 SINGLE_INPUT_CLUSTER_DEVICE_CLASS = {
     # this works for now but if we hit conflicts we can break it out to
     # a different dict that is keyed by manufacturer
     SMARTTHINGS_ACCELERATION_CLUSTER: BINARY_SENSOR,
     SMARTTHINGS_HUMIDITY_CLUSTER: SENSOR,
-    zcl.clusters.closures.DoorLock: LOCK,
-    zcl.clusters.closures.WindowCovering: COVER,
+    zcl.clusters.closures.DoorLock.cluster_id: LOCK,
+    zcl.clusters.closures.WindowCovering.cluster_id: COVER,
     zcl.clusters.general.AnalogInput.cluster_id: SENSOR,
     zcl.clusters.general.MultistateInput.cluster_id: SENSOR,
-    zcl.clusters.general.OnOff: SWITCH,
-    zcl.clusters.general.PowerConfiguration: SENSOR,
-    zcl.clusters.homeautomation.ElectricalMeasurement: SENSOR,
-    zcl.clusters.hvac.Fan: FAN,
-    zcl.clusters.measurement.IlluminanceMeasurement: SENSOR,
-    zcl.clusters.measurement.OccupancySensing: BINARY_SENSOR,
-    zcl.clusters.measurement.PressureMeasurement: SENSOR,
-    zcl.clusters.measurement.RelativeHumidity: SENSOR,
-    zcl.clusters.measurement.TemperatureMeasurement: SENSOR,
-    zcl.clusters.security.IasZone: BINARY_SENSOR,
-    zcl.clusters.smartenergy.Metering: SENSOR,
+    zcl.clusters.general.OnOff.cluster_id: SWITCH,
+    zcl.clusters.general.PowerConfiguration.cluster_id: SENSOR,
+    zcl.clusters.homeautomation.ElectricalMeasurement.cluster_id: SENSOR,
+    zcl.clusters.hvac.Fan.cluster_id: FAN,
+    zcl.clusters.measurement.IlluminanceMeasurement.cluster_id: SENSOR,
+    zcl.clusters.measurement.OccupancySensing.cluster_id: BINARY_SENSOR,
+    zcl.clusters.measurement.PressureMeasurement.cluster_id: SENSOR,
+    zcl.clusters.measurement.RelativeHumidity.cluster_id: SENSOR,
+    zcl.clusters.measurement.TemperatureMeasurement.cluster_id: SENSOR,
+    zcl.clusters.security.IasZone.cluster_id: BINARY_SENSOR,
+    zcl.clusters.smartenergy.Metering.cluster_id: SENSOR,
 }
 
 SINGLE_OUTPUT_CLUSTER_DEVICE_CLASS = {zcl.clusters.general.OnOff: BINARY_SENSOR}
```

Comment:
since the other was changed to be by cluster id and not class shouldn't we change this one too to keep them consistent?

Reply:
yep. will do.

Reply:
Done.